src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -32,6 +33,7 @@
 
 	private int enemyCount;
 	public static final int MAX_ENEMY_COUNT = 10;
+	private EnemyFishFactory factory = new EnemyFishFactory();
```

Comment:
You should add a getter and setter for the factory, so that tests can mock it, and MenuField doesn't need one.

<interface